bridgetown-website/netlify.toml

```diff
@@ -0,0 +1,40 @@
+[build]
+  command = "yarn deploy && ./bin/netlify.sh"
+  publish = "output"
+
+[build.processing]
+  skip_processing = false
+[build.processing.css]
+  bundle = false
+  minify = false
+[build.processing.js]
+  bundle = false
+  minify = false
+[build.processing.html]
+  pretty_urls = true
+[build.processing.images]
+  compress = false
+
+[[headers]]
+  for = "*"
+  [headers.values]
+    X-Frame-Options = "DENY"
+    X-XSS-Protection = "1; mode=block"
+    X-Content-Type-Options = "nosniff"
+    Strict-Transport-Security = "max-age=15552000; includeSubDomains"
+    Referrer-Policy = "no-referrer-when-downgrade"
+    Cache-Control = "public, max-age=604800, s-max-age=604800"
+
+[[headers]]
+  for = "/"
+  [headers.values]
+    Link = [
+      "<CSS_PATH>; rel=preload; as=style",
+      "<JS_PATH>; rel=preload; as=script",
+      "<https://NETLIFY_IMAGES_CDN_DOMAIN>; rel=preconnect"
+    ]
+
+[[headers]]
+  for = "/*.(png|jpg|js|css|svg|woff|ttf|eot|ico|woff2)"
+  [headers.values]
+    Cache-Control = "public, max-age=31536000, s-max-age=31536000"
```

Comment:
Things like fonts/images aren't likely to change, so 1 year is an acceptable expiry time.